Solutions/Menes.Abstractions/Menes/IOpenApiLinkOperationMap.cs

```diff
@@ -49,6 +49,6 @@ public interface IOpenApiLinkOperationMap
         /// <param name="relationType">The relation type of the link to be generated.</param>
         /// <param name="context">A user-defined context in which the link is being used. This allows you to discriminate between operations with the same content type and relation, targetting different operations.</param>
         /// <param name="operationId">The Id of the operation that will be used to resolve the link.</param>
-        void Map<T>(string relationType, string context, string operationId);
+        void MapByContentTypeRelationContextAndOperationId<T>(string relationType, string context, string operationId);
```

Comment:
I'm wondering if these names might be a bit more legible if you put `And` between each "thing" in the name. It makes it more of a mouthful but less ambiguous - e.g. this would become `MapByContentTypeAndRelationAndContextAndOperationId` but at least there's less chance of someone asking what a `RelationContext` is.

Reply:
Agreed. Done.